_includes/contact_form.html

```diff
@@ -23,229 +18,229 @@
   let topicMap = [
     new TopicEntry(
       'Create Account',
-      "{{ site.data.[page.lang].settings.contact_page.support_request_form.topics.creating_account }}",
+      '{{ site.data.[page.lang].settings.contact_page.support_request_form.topics.creating_account | escape_quotes }}',
       [
         new TopicEntry(
           'Email delivery issue',
-          "{{ site.data.[page.lang].settings.contact_page.support_request_form.subtopics.email_delivery_issue }}",
+          '{{ site.data.[page.lang].settings.contact_page.support_request_form.subtopics.email_delivery_issue | escape_quotes}}',
```

Comment:
Nit: Would be good to have balanced whitespace in the tag, consistent with the one above.

```suggestion
          '{{ site.data.[page.lang].settings.contact_page.support_request_form.subtopics.email_delivery_issue | escape_quotes }}',
```

(Also applies throughout the file)

Reply:
Fixed. Will rebuild and retest and merge!